testcases/multizone_residential_hydronic/models/find_days.py

```diff
@@ -0,0 +1,14 @@
+'''
+Finds the peak and typical heat days for the multizone_residential_hydronic.
+This case needs to be deployed if using BOPTEST image.
+
+'''
+
+from data.find_days import find_days
+import json
+
+days = find_days(heat='sum_Q_flow', cool=None,
```

Comment:
@dhblum we leave out cooling because it is idealized right?

Reply:
Yes that was my thinking.  Does that make sense?  Maybe we discuss what to do with cooling in a future WP1.2 meeting?

Reply:
I think that makes sense. Still may be worth discussing this in a WP1.2 meeting.

Reply:
Ok yes.